### ISSUE TITLE
Example Mat folder

### ISSUE THREAD

**Issue by Cyclic** [lai001/VideoClap#1]
Just checking out your project. Pretty cool! Is there any chance that you can post the Mat folder somewhere to test this out?

Thank you!

**Comment by lai001** [comment]
No, but i will consider using UIImagePickerController to add image or video for testing. To test animated sticker, you can use these json files in here [https://github.com/airbnb/lottie-ios/tree/master/Example/lottie-swift/TestAnimations](url)
